sark/sark_isr.c

```diff
@@ -1,12 +1,22 @@
 //------------------------------------------------------------------------------
-//
-// sark_isr.c       Interrupt handlers for SARK
-//
-// Copyright (C)    The University of Manchester - 2009-2013
-//
-// Author           Steve Temple, APT Group, School of Computer Science
-// Email            temples@cs.man.ac.uk
-//
+//! \file
+//! \brief     Interrupt handlers for SARK
+//!
+//! \copyright &copy; The University of Manchester - 2009-2019
+//!
+//! \author    Steve Temple, APT Group, School of Computer Science
+//!
+//! \details
+//! In general, for the classes of interrupt with in this code, the options
+//! for handling are:
+//!
+//! 1. to dump the interrupt (clear the hardware but otherwise do nothing
+//! 2. to process the interrupt as a fast interrupt, using the FIQ (a very
+//!    limited number of event types can use this at a time)
+//! 3. to process the interrupt immediately as a standard interrupt.
+//! 4. to clear the hardware rapidly and enqueue a user-level callback for
```

Comment:
Nothing that needs changing.  Just like the word "enqueue", which I haven't seen used before IRL.